gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -315,6 +315,7 @@ main (int argc, char *argv[])
   /* Upstream has "existing user" mode for new user accounts. In Endless, we
      skip straight to the welcome tutorial instead. */
   if (get_mode () == GIS_DRIVER_MODE_EXISTING_USER) {
+    setlocale (LC_ALL, "");
```

Comment:
In retrospect I would have preferred it if we modified the `GisDriver` to run with this new behavior when in existing user mode, instead of doing it behind its back, but I guess it's too late for that now. We can revisit that the next time we rebase on a new g-i-s.

However, it seems more robust to at least move this whole block after the GTK initialization.